docs/project/reward-algorithm.mdx

```diff
@@ -5,52 +5,48 @@ sidebar_label: Reward Algorithm
 slug: /reward-algorithm
 ---
 
-# Reward Algorithm
-
 Rewarding the members of our network for sharing their local weather data is a crucial part of our system (**currently on testnet**).
 
-Our **Reward Algorithm** is a unique, fair and transparent algorithm that will handle how rewards are being distributed to the network members.
+Our **Reward Algorithm** is a unique, fair, and transparent algorithm that will handle how rewards are being distributed to the network members.
 
-We are currently on the first iteration of that algorithm that has been implemented in a basic way that takes into account only
-if any given weather station is connected or not in our network.
+We are currently on the first iteration of that algorithm, which has been implemented in a basic way—only taking into account whether or not a station is connected to the network.
 
-There are 3 steps on the algorithm:
+The algorithm is comprised of three steps:
   - The minting of the WXM Testnet tokens
   - The validation of the weather stations and their weather data
   - The reward distribution
 
 
 :::note Important
-Connecting a wallet is mandatory for getting rewards. If a user has not connected a wallet, the rewards will be lost.
+Connecting a wallet is mandatory to receive rewards. If a user has not connected a wallet, the rewards will be lost.
 :::
 
-## How Often does the Reward Algorithm runs?
-It runs daily for the day before.
+## How Often does the Reward Algorithm run?
+The algorithm runs every day and takes into account the data for the previous day.
```

Comment:
Let's be more descriptive: `runs every day for rewarding weather stations that were actively sending data to the network for the previous day`?